tests/main/views/test_suppliers.py

```diff
@@ -461,7 +461,9 @@ def test_update_response_matches_payload(self):
         supplier.pop('links')
         supplier.pop('id')
 
-        assert supplier == payload
+        payload_with_defaults = {**payload, 'companyDetailsConfirmed': False}
+
+        assert supplier == payload_with_defaults
```

Comment:
how does `'companyDetailsConfirmed': False` get into the supplier returned in the response in this test? I'm assuming it's the supplier that's POSTed on line 403. But that's loaded from the example listing from `example_listings/supplier_creation.json` which doesn't have the new `companyDetailsConfirmed ` key in it?

Reply:
It's a default value - everyone get's False to start with.

Reply:
Ah, because the migrations get run before the tests.